openedx/core/djangoapps/heartbeat/default_checks.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+from django.core.cache import cache
+from django.db import connection
+from django.db.utils import DatabaseError
+from time import sleep, time
+from xmodule.modulestore.django import modulestore
+from xmodule.exceptions import HeartbeatFailure
+
+from .tasks import sample_task
+
+
+#DEFAULT SYSTEM CHECKS
+
+#Modulestore
+
+def check_modulestore(request):
+    # This refactoring merely delegates to the default modulestore (which if it's mixed modulestore will
+    # delegate to all configured modulestores) and a quick test of sql. A later refactoring may allow
+    # any service to register itself as participating in the heartbeat. It's important that all implementation
+    # do as little as possible but give a sound determination that they are ready.
+    try:
+        #@TODO Do we want to parse the output for split and mongo detail and return it?
+        modulestore().heartbeat()
+        return 'modulestore', True, "OK"
+    except HeartbeatFailure as fail:
+        return 'modulestore', False, unicode(fail)
```

Comment:
It doesn't really matter for Python 2, but we're returning different types in the success and fail cases: the third string is a bytestring on success, and unicode on failure.